pkg/scheduler/framework/runtime/framework.go

```diff
@@ -526,11 +526,9 @@ func (f *frameworkImpl) expandMultiPointPlugins(logger klog.Logger, profile *con
 		// - part 3: other plugins (excluded by part 1 & 2) in regular extension point.
 		newPlugins := reflect.New(reflect.TypeOf(e.slicePtr).Elem()).Elem()
 		// part 1
-		for _, name := range enabledSet.list {
```

Comment:
cc @Huang-Wei 
Any purpose of iterating through enabledSet instead of overrodePlugins in #108613?

Reply:
Traversing `enabledSet` instead of `overridePlugins` directly is to allow the plugins in `overridePlugins` to maintain their original order in `enabledSet` when they were added to the `newPlugins`.

Reply:
Thanks for your information. I think it would be better to iterate over a copy of `enabledSet` than iterate over `overridePlugins`. After testing, this method can also solve the problem of plugins being loaded multiple times.

Reply:
@caohe we still need a unit test to cover this, can you add a test case in `pkg/scheduler/framework/runtime/framework_test.go#TestNewFrameworkMultiPointExpansion`? After addint a test case, we are able to make sure this bug can be reproduced and fixed.

Reply:
Yes, I'm doing this.